CONTRIBUTING.md

```diff
@@ -93,7 +93,7 @@ helm dependency update
 Minimally:
 
 ```
-helm install charts/argo-cd --namespace argocd -n argo-cd
+helm install argocd  argo/argo-cd  -n argocd --create-namespace
```

Comment:
Oh didnt saw the PR in time. `charts/argo-cd` was correct since this file describes howto test changes before contributing/opening a PR :(

FYI @jmeridth @fengshunli